src/iexamine.cpp

```diff
@@ -3962,18 +3962,20 @@ void iexamine::autodoc( player &p, const tripoint &examp )
     }
 
     bool needs_anesthesia = true;
-    std::vector<item_comp> acomps;
+    std::vector< item *> anesth_kit;
```

Comment:
Why are you not using a `tool_comp` here?

Reply:
Done